src/components/AddPlaidBankAccount.js

```diff
@@ -249,6 +249,7 @@ function AddPlaidBankAccount({
                     height={iconSize}
                     width={iconSize}
                     additionalStyles={iconStyles}
+                    fill={theme.icon}
```

Comment:
This created a regression due to expo https://github.com/Expensify/App/issues/33664